RELEASING.md.template

```diff
@@ -1,11 +1,13 @@
 # Releasing
 
 1. Update version file accordingly.
+1. If necessary, install dependencies and update any version snapshot files (`Gemfile.lock`, etc)
```

Comment:
This phrasing feels awkward. It's also maybe entirely Ruby specific. Is calling this out necessary? Is there a better way to phrase this?